### ISSUE TITLE
Cellular Data

### ISSUE THREAD

**Issue by Seferi** [scheich/Throughput#1]
Hi, it works on Wi-Fi but when i switch to cellular data it stops working. And when i switch back to Wi-Fi it doesn't start..

**Comment by Seferi** [comment]
On Android 7.1.1 LineageOS

**Comment by scheich** [comment]
Do you have given all app permissions. And I've not tested it yet with Android 7

**Comment by Seferi** [comment]
Yes, except location.. I didn't see why it would need that permission...

**Comment by scheich** [comment]
Try turning it on for a test. At the moment there is no error handling for that missing permission. I think the right is needed to get the signal strength or the neighbour cell count.

**Comment by Seferi** [comment]
Enabling location access solves the problem mate. Thanks..
